lib/happo/public/happo-runner.js

```diff
@@ -6,11 +13,11 @@ window.happo = {
   currentRenderedElement: undefined,
   errors: [],
 
-  define: function(description, func, options) {
+  define: function define(description, func, options) {
```

Comment:
Indeed, but it is better because these function names will show up in stack traces and profiling. We could make this nicer if we used object shorthand syntax, but we'd need to transpile this then.